opentelemetry-sdk/src/trace/tracer.rs

```diff
@@ -120,7 +120,7 @@ impl Tracer {
     }
 }
 
-static EMPTY_ATTRIBUTES: Lazy<OrderMap<Key, Value>> = Lazy::new(Default::default);
+static EMPTY_ATTRIBUTES: Lazy<Vec<KeyValue>> = Lazy::new(Default::default);
```

Comment:
I wonder if we even need this as `static` seeing as how it's pretty cheap to create an empty `Vec`.  In fact, `Vec::new` is a `const fn`.

Reply:
makes sense. Tacking this in the next PR continuing this direction!